argilla/pyproject.toml

```
@@ -13,6 +13,7 @@ dynamic = ["version"]
 dependencies = [
     "httpx>=0.26.0",
     "pydantic>=2.6.0, <3.0.0",
+    "huggingface_hub>=0.0.12",
```

```suggestion
    "huggingface_hub>=0.22.0",
```

`0.0.12` is a veeeery old release :smile:. Latest is 0.23.4 (soon 0.24.x). I would advice to require at least `0.22.0` which includes quite a lot of bug fixes.